submit-api/src/submit_api/services/item.py

```diff
@@ -11,3 +11,16 @@ def get_item_by_id(cls, item_id):
         """Get item by id."""
         item = ItemModel.find_by_id(item_id)
         return item
+
+    @classmethod
+    def update_item(cls, item_id, update_data):
+        """Update item by id."""
+        item = cls.get_item_by_id(item_id)
```

Comment:
just for clarity item -> submission_item, because we are using the dict.items() method and devs might be confused